pygsp/graphs/graph.py

```diff
@@ -187,7 +187,8 @@ def set_coordinates(self, kind='spring', **kwargs):
             nodes when plotting the graph. Can either pass an array of size Nx2
             or Nx3 to set the coordinates manually or the name of a layout
             algorithm. Available algorithms: community2D, random2D, random3D,
-            ring2D, line1D, spring. Default is 'spring'.
+            ring2D, line1D, spring, laplacian_eigenmap2D, laplacian_eigenmap3D.
```

Comment:
Would be nice to have an `n_dimensions` parameter here instead of `laplacian_eigenmap2D`, `laplacian_eigenmap3D`. User can then do an eigenmap in n dimensions. Should also convert the others. But I won't object here, as this issue predates that PR.

Reply:
Sure, no prob.